src/modules/itslearning/event-handlers/itslearning-persons.event-handler.ts

```diff
@@ -43,69 +43,82 @@ export class ItsLearningPersonsEventHandler {
     @EventHandler(PersonRenamedEvent)
     public async personRenamedEventHandler(event: PersonRenamedEvent): Promise<void> {
         await this.personUpdateMutex.runExclusive(async () => {
-            this.logger.info(`Received PersonRenamedEvent, ${event.personId}`);
+            this.logger.info(`[EventID: ${event.eventID}] Received PersonRenamedEvent, ${event.personId}`);
 
             if (!this.ENABLED) {
-                return this.logger.info('Not enabled, ignoring event.');
+                return this.logger.info(`[EventID: ${event.eventID}] Not enabled, ignoring event.`);
             }
 
             if (!event.referrer) {
-                return this.logger.error(`Person with ID ${event.personId} has no username!`);
+                return this.logger.error(
+                    `[EventID: ${event.eventID}] Person with ID ${event.personId} has no username!`,
+                );
             }
 
             const readPersonResult: Option<PersonResponse> = await this.itslearningPersonRepo.readPerson(
                 event.personId,
+                `${event.eventID}-PERSON-EXISTS-CHECK`,
```

Comment:
Is it possible to put this name + formatting in the base-event, so it is centralized?

Reply:
I feel like this is more of a specific for the event-handler. The event itself has an ID, but every event might send multiple requests to itslearning. Because of this, I concatenate this here.